pkg/controller/constraint/constraint_controller.go

```diff
@@ -511,6 +468,16 @@ func (r *ReconcileConstraint) getOrCreatePodStatus(ctx context.Context, constrai
 	return statusObj, nil
 }
 
+func isRegoOnlyCT(ct *v1beta1.ConstraintTemplate) bool {
```

Comment:
More appropriately, this should be whether the CT has VAPCel in it. Otherwise, this function name may make no sense if other languages get added later.